packages/feedback-spa/src/app/feedback/feedback.service.ts

```diff
@@ -46,10 +46,10 @@ export class FeedbackService extends GraphQLModule {
    * Get feedback by id
    */
 
-  getFeedback(_id: string) {
+  getFeedback(id: string) {
     return this.apollo.watchQuery<FeedbackResponse>({
       variables: {
-        _id
+        _id: id
```

Comment:
The variable name should be the same as in the gql query.
```suggestion
        id: id
```